src/JDS.jl

```diff
@@ -7,6 +7,21 @@ import Makie
 using Reexport: @reexport
 
 @reexport begin
+    using AlgebraOfGraphics:
+        data,
+        density,
+        draw,
+        draw!,
+        expectation,
+        frequency,
+        legend!,
+        linear,
+        mapping,
+        nonnumeric,
+        renamer,
+        smooth,
+        sorter,
+        visual
```

Comment:
Ah. Nice that you explicitly imported all the symbols. That must have been a lot of work. It will hopefully save us some headaches later when two packages export similar names :+1:

Reply:
Yeah that happens in AoG a lot!